docs/niswitch/attributes.rst

```diff
@@ -96,10 +81,7 @@ niswitch.Session properties
 
 .. py:attribute:: cabled_module_scan_advanced_bus
 
-    This property has been deprecated and might be removed from a future
-    release of NI-SWITCH. Use `niSwitch Route Scan Advanced
-    Output <switchviref.chm::/niSwitch_Route_Scan_Advanced_Output.html>`__
-    VI instead.
+    This attribute has been deprecated and may be removed from a future release of  NI-SWITCH.  Use the niSwitch_RouteScanAdvancedOutput function instead.
```

Comment:
Looks like there are a few deprecated (per documentation) attributes here.

Reply:
Yes, we need to audit all attributes for all drivers. Issue #562